### PR TITLE
Fix Advanced Language expressions Reporting

### PR DESCRIPTION
**Pull request by owenrumney**

### CODE REVIEW

File: test-cases/terraform/hcl_language_complexity/using_count_and_ternary_expr/main.tf [hunk 1]
```diff
@@ -30,3 +30,38 @@ resource "aws_iam_user_login_profile" "jill" {
   user    = aws_iam_user.jill.name
   pgp_key = "keybase:some_person_that_exists"
 }
+
+resource "aws_security_group_rule" "trust-rules-dev" {
```
Comment (yi2020) [inline test-cases/terraform/hcl_language_complexity/using_count_and_ternary_expr/main.tf:34]: Why did you add these? The goal of this test case is to see if the tool can capture the use of count and a ternary expression.

File: test-cases/terraform/hcl_language_complexity/using_count_and_ternary_expr/tfsec_results.txt [hunk 2]
```diff
@@ -1,24 +1,73 @@
 
-  times
-  ------------------------------------------
-  disk i/o             51.157832ms
-  parsing HCL          806.083µs
-  evaluating values    4.794334ms
-  running checks       11.739791ms
+[0m[31m[1m  [1m[4mResult 1[0m[31m[1m
```
Comment (yi2020) [inline test-cases/terraform/hcl_language_complexity/using_count_and_ternary_expr/tfsec_results.txt:2]: Can you run tfsec with no color to avoid these VTERM markings ([0m etc)

File: test-cases/terraform/hcl_language_complexity/using_locals/tfsec_results.txt [hunk 3]
```diff
@@ -1,24 +1,135 @@
 
-  times
-  ------------------------------------------
-  disk i/o             57.797751ms
-  parsing HCL          532.041µs
-  evaluating values    3.101918ms
-  running checks       11.189583ms
+[0m[31m[1m  [1m[4mResult 1[0m[31m[1m
+[0m[0m
+  [34m[[39maws-s3-enable-versioning[34m][39m[34m[[39m[0m[33mMEDIUM[39m[0m[34m][39m Resource 'aws_s3_bucket.access-logs-bucket[0]' has versioning block but is disabled
```
Comment (yi2020) [inline test-cases/terraform/hcl_language_complexity/using_locals/tfsec_results.txt:4]: if versioning_enabled was "true", tfsec wouldn't alert?

File: test-cases/terraform/hcl_language_complexity/using_module_multi/mymodule/bucket.tf [hunk 4]
```diff
@@ -0,0 +1,5 @@
+resource "aws_s3_bucket" "cloudtrail_logs" {
```
Comment (yi2020) [inline test-cases/terraform/hcl_language_complexity/using_module_multi/mymodule/bucket.tf:1]: Can you help me understand why this file was added?

**Comment by yi2020** [comment]
@owenrumney sorry for the delay, please take a look at my comments

File: test-cases/terraform/hcl_language_complexity/using_count_and_ternary_expr/main.tf (hunk 1 above)
Comment (owenrumney) [reply test-cases/terraform/hcl_language_complexity/using_count_and_ternary_expr/main.tf:34]: Because tfsec doesn't support the issue you used in the example - the reason being that we don't consider that to be a valid check. It does however support counts, so needed an example block that would demonstrate that.

**Comment by owenrumney** [comment]
I think this whole repo is very disingenuous tbh - so I'll close this PR and move on.

**Comment by yi2020** [comment]
That is not my intention @owenrumney . I'm happy to help get your PR through, I'm trying to understand what was done there. The advanced expressions were added due to feedback I received from TF users. If it doesn't reflect on tfsec correctly, happy to remediate that. What needs to be fixed?
